program/src/state.rs

```diff
@@ -601,7 +608,28 @@ pub struct Validator {
     #[serde(serialize_with = "serialize_b58")]
     pub fee_address: Pubkey,
 
-    /// Start (inclusive) of the seed range for currently active stake accounts.
+    /// Seeds for active stake accounts.
+    pub stake_seeds: SeedRange,
+    /// Seeds for inactive stake accounts.
+    pub unstake_seeds: SeedRange,
+
+    /// Sum of the balances of the stake accounts.
```

Comment:
s/of the stake accounts/of the stake accounts and unstake accounts/, to clarify that it counts both.